src/FINDOLOGIC/Api/Definitions/QueryParameter.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace FINDOLOGIC\Api\Definitions;
+
+class QueryParameter
+{
+    const
```

Comment:
I don't see `outputAttrib` anywhere. Is this intentional?

Reply:
Yes but I will discuss this with potential users of the library, so we see if it makes sense and also if this parameter is "officially" supported.

Reply:
[outputAttrib](https://docs.findologic.com/doku.php?id=integration_documentation:request&s[]=outputattrib) is documented officially. It doesn't have effect for XML response, but once we have a JSON response, it will make a lot of sense to have it.